platform/ios/src/MGLMapView.mm

```diff
@@ -1718,9 +1718,12 @@ - (void)handleRotateGesture:(UIRotationGestureRecognizer *)rotate
     if ( ! self.isRotateEnabled) return;
 
     if (MGLDegreesFromRadians(self.rotationBeforeThresholdMet) < self.rotationThresholdWhileZooming && self.isZooming) {
+        _changeDelimiterSuppressionDepth++;
```

Comment:
What is the purpose of `changeDelimiterSuppressionDepth`? Should it be iterated if there’s no rotation/change happening yet?

Reply:
I found another way to address the exception when `_changeDelimiterSuppressionDepth <= 0`. Even if the gesture is delayed, the gesture's state is only equal to `UIGestureRecognizerStateBegan` once. By returning, I was skipping the setup that happens in this method.